openbb_platform/platform/provider/openbb_provider/standard_models/sector_pe.py

```diff
@@ -0,0 +1,26 @@
+"""Sector P/E Ratio data model."""
+
+from datetime import date as dateType
+from typing import Optional
+
+from pydantic import Field
+
+from openbb_provider.abstract.data import Data
+from openbb_provider.abstract.query_params import QueryParams
+
+
+class SectorPeQueryParams(QueryParams):
```

Comment:
Same as with IndexPe. Can we do `SectorPE` here?

Reply:
For some additional context, `FMPETFNAVQueryParams`, would be difficult to read vs. `FmpEtfNavQueryParams`

Reply:
Yes it would.
Special cases aren't special enough to break the rules, although practicality beats purity